webpack/js/components.js

```diff
@@ -178,6 +178,20 @@ export const App = {
     const BASE_URL = 'https://raw.githubusercontent.com/creativecommons/ccos-scripts/master/normalize_repos'
     const FILE_URL = name => `${BASE_URL}/${name}.json`
 
+    const octokit = new Octokit();
+    octokit.search.issuesAndPullRequests({
+      q: 'org:creativecommons is:open is:issue label:"help wanted"',
```

Comment:
Should we remove the issues awaiting triage?

Reply:
I think we should add a separate filter for issues awaiting triage.